MetaMask/metamask-extension
# Fix #20274 - Update locked network menu controls

## Explanation

* Fixes #20274

This PR ensures (1) the user doesn't see the "delete network" icon when MetaMask is locked, and (2) the user doesn't see the "Add Networks" button when locked.

## Manual Testing Steps

1. Lock MetaMask
2. See the "Add network" button disabled
3. See no delete icon when hovering over custom network

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
